emukit/core/acquisition.py

```diff
@@ -66,6 +66,12 @@ def evaluate_with_gradients(self, x: np.ndarray) -> Tuple[np.ndarray, np.ndarray
         """
         raise NotImplementedError("Gradients not implemented for this acquisition function")
 
+    def update_between_iterations(self) -> None:
```

Comment:
Is it usual to call methods by the way they are used in some application rather than what they do?

Reply:
I second this comment. "iteration" concept does not exist on acquisition on CandPointCalc level. This needs rephrasing to reflect what this method actually does.

Also, I find it weird this method does not take any inputs. Even though ES case does not need it, I wander if it is an update at all. Perhaps ES case is so special it could be re-implemented slightly to avoid this new API altogether?

Reply:
I'll rename as `update_parameters` and add loop state as input